test/service/test_response.py

```diff
@@ -1149,6 +1156,84 @@ def test_project_accessions_response(self):
         }
         self.assertElasticsearchResultsEqual(keyword_response, expected_response)
 
+    def test_cell_suspension_response(self):
+        """
+        This method tests the KeywordSearchResponse object for the projects entity type,
+        specifically making sure the selectedCellType field is present with the correct value
+        """
+        keyword_response = KeywordSearchResponse(
+            hits=self.get_hits('projects', '250aef61-a15b-4d97-b8b4-54bb997c1d7d'),
+            entity_type='cell_suspensions'
+        ).return_response().to_json()
+        expected_response = {
+            "hits": [
+                {
+                    "protocols": [
+                        {
+                            "libraryConstructionApproach": ["MARS-seq"],
+                            "instrumentManufacturerModel": ["Illumina NextSeq500"]
+                        }
+                    ],
+                    "entryId": "250aef61-a15b-4d97-b8b4-54bb997c1d7d",
+                    "projects": [
+                        {
+                            "projectTitle": "Bone marrow plasma cells from hip replacement surgeries",
+                            "projectShortname": "BM_PC",
+                            "laboratory": [
+                                "Prof. Ido Amit",
+                                "Human Cell Atlas Data Coordination Platform"
+                            ]
+                        }
+                    ],
+                    "specimens": [
+                        {
+                            "id": ["Hip9_specimen"],
+                            "genusSpecies": ["Homo sapiens"],
+                            "organ": ["hemopoietic organ"],
+                            "organPart": [
+                                "bone marrow",
+                                "temporal lobe"
+                            ],
+                            "organismAge": ["76"],
+                            "organismAgeUnit": ["year"],
+                            "biologicalSex": ["male"],
+                            "disease": [],
+                            "preservationMethod": [None],
+                            "source": [
+                                "specimen_from_organism",
+                                "cell_suspension",
+                                "donor_organism"
+                            ]
+                        }
+                    ],
+                    "cellSuspensions": [
+                        {
+                            "organ": ["hemopoietic organ"],
+                            "organPart": [
+                                "bone marrow",
+                                "temporal lobe"
+                            ],
+                            "selectedCellType": ["Plasma cells"],
```

Comment:
Please assert this more surgically. The rest of the literal is inessential for this particular test case. We don't need to assert the entire response literally if the only thing we're looking for is the presence and value of a single property in the response.